tests/IssuanceInitiation.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { AuthzFlowType, convertJsonToURI, convertURIToJsonObject } from '../lib';
+import { IssuanceInitiation } from '../lib';
```

Comment:
It was suggested to rename the source file. If the source file is renamed. then this must also be renamed.

Reply:
Nah. For an external Developer having a single class about IssuanceInitiation makes way more sense